src/include/sof/drivers/ssp.h

```diff
@@ -154,6 +154,12 @@ extern const struct dai_driver ssp_driver;
 #define SSP_REG_MAX	SSIOC
 #endif
 
+/* SSTSA bits */
+#define SSTSA_TXEN		BIT(8)
+
+/* SSRSA bits */
+#define SSRSA_RXEN		BIT(8)
```

Comment:
I assume there are no other bits? I usually put even those that I won't really use in the driver if the hardware offers them but I'll leave this up to you to decide.

Reply:
Now there are all

Reply:
Perfect :)